### PR TITLE
Crab game

### PR DESCRIPTION
**Pull request by joeyshi12**
unknown

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 from ShapeDetector import paint
+# @Copyright 2021 joeyshi.tech
```
Comment (102plee) [inline main.py:2]: Please consider changing this to a copyright for 102peterlee.me

**Review by 102plee — CHANGES_REQUESTED** [review]
Just a couple reviews for you to fix up before merging

File: main.py (hunk 1 above)
Comment (joeyshi12) [reply main.py:2]: PR is already growing too large, left as a todo here https://github.com/102plee/AI_Projects/issues/2

Comment (102plee) [reply main.py:2]: I'm afraid this PR can't be merged without addressing this pressing issue

**Review by 102plee — APPROVED** [review]

**Pull request merged**